boards/arm/mimxrt595_evk/mimxrt595_evk_cm33-pinctrl.dtsi

```diff
@@ -24,6 +24,23 @@
 		};
 	};
 
+	pinmux_flexcomm4_i2c: pinmux_flexcomm4_i2c {
+		group0 {
+			pinmux = <FC4_TXD_SCL_MISO_WS_PIO0_29>,
+			       <FC4_RXD_SDA_MOSI_DATA_PIO0_30>;
+			input-enable;
+			slew-rate = "normal";
+			drive-strength = "high";
+			drive-open-drain;
+		};
+		group1 {
+			pinmux = <GPIO_PIO022_PIO0_22>;
+			input-enable;
+			slew-rate = "normal";
+			drive-strength = "normal";
+		};
```

Comment:
why is this needed? sensor is configuring GPIO using the GPIO API.

Reply:
It's indeed not needed. Removed it. Thanks.